core/object/class_db.h

```diff
@@ -288,46 +292,65 @@ class ClassDB {
 		if constexpr (std::is_same<typename member_function_traits<M>::return_type, Object *>::value) {
 			bind->set_return_type_is_raw_object_ptr(true);
 		}
-		return bind_methodfi(METHOD_FLAGS_DEFAULT, bind, p_method_name, sizeof...(p_args) == 0 ? nullptr : (const Variant **)argptrs, sizeof...(p_args));
+		return bind_methodfi(METHOD_FLAGS_DEFAULT, bind, false, p_method_name, sizeof...(p_args) == 0 ? nullptr : (const Variant **)argptrs, sizeof...(p_args));
 	}
 
+	template <class N, class M, typename... VarArgs>
+	static MethodBind *bind_compatibility_method(N p_method_name, M p_method, VarArgs... p_args) {
```

Comment:
Maybe that train has sailed, but maybe there's a good way of sharing code at least between regular/compat versions of the functions, given that the only difference is a `true`/`false`. A way which is not a macro, I mean!

Something like this (not tried):
```
	template <class N, class M, typename... VarArgs>
	static MethodBind *_base_bind_method(bool p_is_compat, N p_method_name, M p_method, VarArgs... p_args) {
		Variant args[sizeof...(p_args) + 1] = { p_args..., Variant() }; // +1 makes sure zero sized arrays are also supported.
		const Variant *argptrs[sizeof...(p_args) + 1];
		for (uint32_t i = 0; i < sizeof...(p_args); i++) {
			argptrs[i] = &args[i];
		}
		MethodBind *bind = create_method_bind(p_method);
		if constexpr (std::is_same<typename member_function_traits<M>::return_type, Object *>::value) {
			bind->set_return_type_is_raw_object_ptr(true);
		}
		return bind_methodfi(METHOD_FLAGS_DEFAULT, bind, p_is_compat, p_method_name, sizeof...(p_args) == 0 ? nullptr : (const Variant **)argptrs, sizeof...(p_args));
	}

        template <class N, class M, typename... VarArgs>
	static MethodBind *bind_method(N p_method_name, M p_method, VarArgs... p_args) {
		return _base_bind_method(false, p_method_name, p_method, p_args...);
	}

        template <class N, class M, typename... VarArgs>
	static MethodBind *bind_compatibility_method(N p_method_name, M p_method, VarArgs... p_args) {
		return _base_bind_method(true, p_method_name, p_method, p_args...);
	}
```